composer.json

```diff
@@ -6,12 +6,12 @@
     ],
     "require": {
         "php": ">=7.4.0",
-        "doctrine/annotations": "^1.8.0",
-        "doctrine/collections": "^1.6.8",
-        "doctrine/dbal": "^2.12.0",
+        "doctrine/annotations": "^1.13.3",
+        "doctrine/collections": "^1.8.0",
+        "doctrine/dbal": "^2.13.9",
         "doctrine/lexer": "^1.1.0",
-        "doctrine/orm": "^2.7.0",
-        "doctrine/persistence": "^2.2.2",
+        "doctrine/orm": "^2.13.3",
+        "doctrine/persistence": "^2.5.5",
```

Comment:
A meaningful update would be to allow `doctrine/persistence` 3 and as `doctrine/dbal` 2 is unsupported.